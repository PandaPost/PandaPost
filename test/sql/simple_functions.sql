\set ECHO none

\i test/pgxntool/setup.sql

SELECT plan(
  2 -- all
);

-- all
SELECT is(
    ndall(eval('[[True, False], [False, False]]'::text))::boolean
    , false
    , $$ndall('[[True, False], [False, False]]')$$
);
--SET client_min_messages=debug;
SELECT is(
    str(ndall(eval('[[True, False], [False, False]]'::text),keepdims:=true))
    , str(eval('[[False]]'))
    , $$ndall('[[True, False], [False, False]]', keepdims := true)$$
);


\i test/pgxntool/finish.sql


-- vi: expandtab sw=2 ts=2

