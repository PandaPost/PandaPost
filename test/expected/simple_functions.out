\set ECHO none
1..2
ok 1 - ndall('[[True, False], [False, False]]')
ok 2 - ndall('[[True, False], [False, False]]', keepdims := true)
# TRANSACTION INTENTIONALLY LEFT OPEN!
